cpu/lm4f120/vectors.c

```diff
@@ -0,0 +1,281 @@
+WEAK_DEFAULT void isr_tim8_up_tim13(void);
+WEAK_DEFAULT void isr_tim8_trg_com_tim14(void);
+WEAK_DEFAULT void isr_tim8_cc(void);
+WEAK_DEFAULT void isr_dma1_stream7(void);
+WEAK_DEFAULT void isr_fsmc(void);
+WEAK_DEFAULT void isr_sdio(void);
+WEAK_DEFAULT void isr_tim5(void);
+WEAK_DEFAULT void isr_spi3(void);
+WEAK_DEFAULT void isr_uart4(void);
+WEAK_DEFAULT void isr_uart5(void);
+WEAK_DEFAULT void isr_tim6_dac(void);
+WEAK_DEFAULT void isr_tim7(void);
+WEAK_DEFAULT void isr_dma2_stream0(void);
+WEAK_DEFAULT void isr_dma2_stream1(void);
+WEAK_DEFAULT void isr_dma2_stream2(void);
+WEAK_DEFAULT void isr_dma2_stream3(void);
+WEAK_DEFAULT void isr_dma2_stream4(void);
+WEAK_DEFAULT void isr_eth(void);
+WEAK_DEFAULT void isr_eth_wkup(void);
+WEAK_DEFAULT void isr_can2_tx(void);
+WEAK_DEFAULT void isr_can2_rx0(void);
+WEAK_DEFAULT void isr_can2_rx1(void);
+WEAK_DEFAULT void isr_can2_sce(void);
+WEAK_DEFAULT void isr_otg_fs(void);
+WEAK_DEFAULT void isr_dma2_stream5(void);
+WEAK_DEFAULT void isr_dma2_stream6(void);
+WEAK_DEFAULT void isr_dma2_stream7(void);
+WEAK_DEFAULT void isr_usart6(void);
+WEAK_DEFAULT void isr_i2c3_ev(void);
+WEAK_DEFAULT void isr_i2c3_er(void);
+WEAK_DEFAULT void isr_otg_hs_ep1_out(void);
+WEAK_DEFAULT void isr_otg_hs_ep1_in(void);
+WEAK_DEFAULT void isr_otg_hs_wkup(void);
+WEAK_DEFAULT void isr_otg_hs(void);
+WEAK_DEFAULT void isr_dcmi(void);
+WEAK_DEFAULT void isr_cryp(void);
+WEAK_DEFAULT void isr_hash_rng(void);
+WEAK_DEFAULT void isr_fpu(void);
+
+/* interrupt vector table */
+ISR_VECTORS const void *interrupt_vector[] = {
+    /* Exception stack pointer */
+    (void*) (&_estack),             /* pointer to the top of the stack */
+    /* Cortex-M4 handlers */
+    (void*) reset_handler_default,  /* entry point of the program */
+    (void*) nmi_default,            /* non maskable interrupt handler */
+    (void*) hard_fault_default,     /* hard fault exception */
+    (void*) mem_manage_default,     /* memory manage exception */
+    (void*) bus_fault_default,      /* bus fault exception */
+    (void*) usage_fault_default,    /* usage fault exception */
+    (void*) (0UL),                  /* Reserved */
+    (void*) (0UL),                  /* Reserved */
+    (void*) (0UL),                  /* Reserved */
+    (void*) (0UL),                  /* Reserved */
+    (void*) isr_svc,                /* system call interrupt, in RIOT used for
+                                     * switching into thread context on boot */
+    (void*) debug_mon_default,      /* debug monitor exception */
+    (void*) (0UL),                  /* Reserved */
+    (void*) isr_pendsv,             /* pendSV interrupt, in RIOT the actual
+                                     * context switching is happening here */
+    (void*) isr_systick,            /* SysTick interrupt, not used in RIOT */
+    // Peripherial interrupts start here.
+    (void *) dummy_handler,         // GPIO Port A                      16
```

Comment:
don't call the dummy_handler here directly. Put in the correct interrupt function names as you defined above. The dummy handler will be called automatically if an interrupt vector is not implemented -> thats why they are decleared to be weak symbols after all...